### DISCUSSION TITLE
Disable/enable Option on condition

### DISCUSSION THREAD

**Discussion by klavsbuss** [newcat/baklavajs#241] [Q&A] [answered]
Hey. I am wondering, is there any way i can change state for Option (disable or hide it) if there is certain condition met in other Option within same node?
Example: If i select V3 in dropdown, i need to disable HD checkbox.  
![image](https://user-images.githubusercontent.com/18184893/210445349-b9a9a145-3989-4cb9-8e28-fc0d24927317.png)

**Comment by newcat** [comment, marked as answer]
Sorry, I overlooked this question. There is no native way to do this as there are also no styles for disabled checkboxes. You could implement a custom option based on the [original implementation](https://github.com/newcat/baklavajs/blob/master/packages/baklavajs-plugin-options-vue/src/CheckboxOption.vue) with the additional styles & logic needed for disabling.
